.gitignore

```diff
@@ -0,0 +1 @@
+./idea/*
```

Comment:
Can you remove this file?

Reply:
can you explain why?
editor configuration directories and files should not be included in a git repository except for some rare cases where you want to have exactly the same ide across multiple developers...

Reply:
or are you talking about the gitignore file in general?
if so please explain why you dont want a gitignore at all? makes no sense to me...

Reply:
I'm fine with a .gitignore file, but it should be consistent with what the project overall needs. I don't believe there's a reason to ignore personal files, such as `./idea/*`. If we needed a `build` directory for this where the build files shouldn't be committed, then that is a valid use of committing this file to the public or if .pyc files (which we should probably add this) need to be ignored.